dashboard/app/controllers/lessons_controller.rb

```diff
@@ -64,9 +64,21 @@ def update
       current_lesson_data[:resources]&.map! {|v| v[:id]}
       old_lesson_data['resources']&.map! {|v| v['id']}
       if old_lesson_data.to_json != current_lesson_data.to_json
+        # Log error details to firehose instead of honeybadger to minimize the
+        # chances of the data being truncated.
+        FirehoseClient.instance.put_record(
+          :analysis,
```

Comment:
What does analysis do? I haven't used that before in firehose

Reply:
the firehose client library in ruby is configured to let you choose which data stream you are logging to: https://github.com/code-dot-org/code-dot-org/blob/d9cd4a19de9c075e972a2c0708e7ffe51435b101/lib/cdo/firehose.rb#L40-L43 

the javascript client library is configured to always go to the "analysis-events" stream: https://github.com/code-dot-org/code-dot-org/blob/ed86d5789abb493a0d56f9e3f2bed466449cdbfe/apps/src/lib/util/firehose.js#L49